### DISCUSSION TITLE
Keeping your battery at low levels

### DISCUSSION THREAD

**Discussion by ghost** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#946] [General]
I was wondering—I see many people keeping their battery levels at lower than 80% most of the time. Makes me wonder why do that? AlDente defaults are to stick with 80%. Why would anyone keep their battery at ~50% most of the time? What are the benefits of doing so?

**Comment by masterzone3d** [comment]
I'm keeping my battery on 50% whole the time due to a full day power adapter usage. 50% is better than 80% so your battery will keep strong for more time :). With Aldente installed my battery stopped to drop. Amazing concept and amazing software.
